lawnchair/src/app/lawnchair/icons/LawnchairIconProvider.kt

```diff
@@ -58,7 +58,7 @@ class LawnchairIconProvider @JvmOverloads constructor(
         lawniconsVersion =
             if (isSupported) context.packageManager.getPackageVersionCode(LAWNICONS_PACKAGE_NAME)
             else 0L
-        _themeMap = if (isSupported) null else DISABLED_MAP
+        _themeMap = if (isSupported && lawniconsVersion in 1..3) null else DISABLED_MAP
```

Comment:
Added this to build ThemedMap only from old version lawnicons and After this pr [815](https://github.com/LawnchairLauncher/lawnicons/pull/815/) Lawnicons will applied based on intent added.

Reply:
Adding this will not apply dynamic calendar for new icon packs.
It's good to add dynamic calendar to icon pack insted of tied up with lawnchair.
Already added as part of PR https://github.com/LawnchairLauncher/lawnicons/pull/815

This fixes an issue that always lawnchair themed calendar will get applied even if we select different icon packs.